lib/mp4/transmuxer.js

```diff
@@ -88,6 +90,55 @@ sumFrameByteLengths = function(array) {
   return sum;
 };
 
+/**
+ * Adjust caption start and end times by the timeline pts values if
+ * keepOriginalTimeStamps is false
+ */
+adjustCaptionTiming = function(caption, timelineStartPts, keepOriginalTimestamps) {
+  var startTime;
+  var endTime;
+
+  if (typeof keepOriginalTimestamps !== 'boolean') {
```

Comment:
It's a bit odd having different behavior depending on whether the value is a boolean or not. Can we have a separate parameter/path if we need it? Or not call into the function if we know of a different case.

Reply:
I can probably remove this with the default value being set now :)